vhdbuilder/packer/components.json

```diff
@@ -301,7 +301,8 @@
       "downloadURL": "mcr.microsoft.com/oss/virtual-kubelet/virtual-kubelet:*",
       "versions": [
         "1.3.6",
```

Comment:
is 1.3.6 still in use?
if not please remove it.

Reply:
1.4.0 is not fully released. let us keep 1.3.6 for now.

Reply:
I do not understand.

the 1.4.0 image is bumped 3 weeks ago. why it's not fully released?